x/staking/src/keeper/unbonding.rs

```diff
@@ -251,48 +272,54 @@ impl<
         &self,
         ctx: &mut CTX,
         validator: &mut Validator,
-    ) -> anyhow::Result<()> {
+    ) {
         if validator.status != BondStatus::Unbonding {
-            return Err(AppError::Custom(format!(
+            panic!(
```

Comment:
Look like this method should return a `Result`?

Reply:
It's unfailable method and it panics. I replaced with assertion.